app/screens/ProductDetailsScreen/ProductDetailsScreen.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Image, Text, SafeAreaView, ScrollView, Button, Alert, RefreshControl } from 'react-native';
+import { Header } from '../../components/Header/Header';
+import { IconBasket } from '../../components/icons/IconBasket';
+import { IconHeart } from '../../components/icons/IconHeart';
+import { IconArrowLeft } from '../../components/icons/IconArrowLeft';
+
+export const ProductDetailsScreen: React.FC = (): JSX.Element => {
+    const [refreshing, setRefreshing] = useState(false);
+    const product = {
+        attributes: {
+            price: '86.99',
+            description: 'Doloremque ab accusamus blanditiis et. Deleniti ma…oloribus ad. Non repellat maiores delectus rerum.',
+            name: 'Raw Edge T Shirt'
+        },
+        id: '16'
+    };
+
+    const wait = (timeout: number) => {
+        return new Promise(resolve => setTimeout(resolve, timeout));
+    }
+
+    const onRefresh = React.useCallback(() => {
+        setRefreshing(true);
+        wait(2000).then(() => setRefreshing(false));
+    }, []);
+
+    return (
+    <SafeAreaView>
+        <ScrollView
+            refreshControl={
+            <RefreshControl
+                refreshing={refreshing}
+                onRefresh={onRefresh}
+            />
+        }>
+            <Header>
+                <View>
+                    <IconArrowLeft fill={'#FFFFFF'} />
+                </View>
+                <View style={styles.headerIcons}>
+                    <IconHeart fill={'#FFFFFF'} />
+                    <View style={styles.headerIconRight}>
+                        <IconBasket fill={'#FFFFFF'} />
+                    </View>
+                </View>
+            </Header>
+            <View style={styles.container}>
+                <View style={styles.imgContainer}>
+                    <Image
+                        style={styles.img}
+                        source={{uri: `https://picsum.photos/id/${product.id}/250`}}
+                        resizeMode={'contain'}
+                    />
+                </View>
+                <View style={styles.cell}>
+                    <Text style={styles.cellValue}>{product.attributes.name}</Text>
+                    <Text style={styles.cellTitle}>{product.attributes.price}</Text>
+                </View>
+                <View style={styles.cell}>
+                    <Text style={styles.cellTitle}>Select color</Text>
+                    <View style={styles.btn}>
```

Comment:
redundant view.

look through your views. Lots of them can be removed. U create too complex tree which leads to the device resources consumption. Remember always that user expects fast rendering and feedback from the application and use as less elements as you can